addons/survey/views/survey_question_views.xml

```diff
@@ -176,7 +176,7 @@
                             </group>
                             <group>
                                 <group string="Display" attrs="{'invisible':[('question_type','not in',['simple_choice', 'multiple_choice'])]}">
-                                    <field name="column_nb" string="Number of columns"/>
+                                    <field name="column_nb" string="Number of columns" attrs="{'invisible':True}"/>
```

Comment:
Actually, can you switch to:

`<field name="column_nb" string="Number of columns" invisible="1"/>`

No real point to using attrs here if you are not evaluating anything.
(Make sure to re-test but it should do exactly the same).

Reply:
Yes it works, thanks